src/components/Profile/Profile.js

```diff
@@ -85,6 +130,7 @@ class UserProfile extends Component {
 
               </Col>
               <Col span={16}>
+              {alertMessage ? <Alert message={alertMessage} type="error" /> : null}
```

Comment:
This alert should display the message sent from the server if status is 200 display success alert else display error alert. If profile is updated successfully then display "Profile Updated Successfully".

Error for No Password and other errors should displayed like 
![Screenshot 2019-05-22 at 10 20 27 PM](https://user-images.githubusercontent.com/31013104/58193205-1c403800-7ce0-11e9-86ba-74949fa81636.png)